README.md

```diff
@@ -693,6 +693,17 @@ Specifies the number of seconds an IP address is in cooldown after pod deletion.
 **Note:** 0 is a supported value, however it is highly discouraged.
 **Note:** Higher cooldown periods may lead to a higher number of EC2 API calls as IPs are in cooldown cache.
 
+----
+
+#### `DISABLE_CONTAINER_V6` (v1.15.0+)
+
+Type: Boolean as a String
+
+Default: `false`
+
+When `DISABLE_CONTAINER_V6` is set, the [tuning plugin](https://www.cni.dev/plugins/current/meta/tuning/) is chained and configured to disable IPv6 networking in each newly created container network namespace. Set this variable when you have an IPv4 cluster and containerized applications that cannot tolerate IPv6 being enabled.
+Container runtimes such as `containerd` will enable IPv6 in newly created container network namespaces regardless of host settings.
```

Comment:
I am not sure how to understand this statement. What is intention for mentioning this information:

> Container runtimes such as `containerd` will enable IPv6 in newly created container network namespaces regardless of host settings.

- Even if we set `DISABLE_CONTAINER_V6=true, the container runtime can enable IPV6 in the newly created namespace?

Reply:
The container runtime, in this case containerd, will create the network namespace and create the loopback interface with an IPv6 address assigned before any CNI plugins run. This statement is to clarify that containerd will not abide by the host settings (host networking sysctls) when it creates the network namespace.

Reply:
VPC CNI is down the chain. So, we're undoing what the CRI did